pages.hi/common/cat.md

```diff
@@ -0,0 +1,23 @@
+# cat
+
+> फ़ाइलों को प्रिंट और संक्षिप्त करें।
+
+- मानक आउटपुट में फ़ाइल की सामग्री प्रिंट करें:
+
+`cat {{फ़ाइल}}`
+
+- लक्ष्य फ़ाइल में कई फ़ाइलों को संयुक्त करें:
+
+`cat {{फ़ाइल1}} {{फ़ाइल2}} > {{लक्ष्य_फ़ाइल}}`
+
+- लक्ष्य फ़ाइल में कई फ़ाइलें जोड़ें:
+
+`cat {{फ़ाइल1}} {{फ़ाइल2}} >> {{लक्ष्य_फ़ाइल}}`
+
+- सभी आउटपुट लाइनों की संख्या:
+
+`cat -n {{फ़ाइल}}`
+
+- गैर-मुद्रण योग्य और व्हाट्सएप वर्ण प्रदर्शित करें (M-उपसर्ग के साथ यदि गैर-ASCII):
```

Comment:
I found `WhatsApp` here instead of `whitespace characters`.
![Screenshot_20201009_111114](https://user-images.githubusercontent.com/1536740/95593535-363b0180-0a20-11eb-880e-13151b33466c.png)